clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/providers/TitusClusterProvider.groovy

```diff
@@ -56,20 +57,25 @@ class TitusClusterProvider implements ClusterProvider<TitusCluster>, ServerGroup
 
   private final Provider<AwsLookupUtil> awsLookupUtil
   private final Provider<CachingSchemaUtil> cachingSchemaUtil
+  private final DefaultAccountCredentialsProvider accountCredentialsProvider
+
 
   @Autowired
   TitusClusterProvider(TitusCloudProvider titusCloudProvider,
                        TitusCachingProvider titusCachingProvider,
                        Cache cacheView,
                        ObjectMapper objectMapper,
                        Provider<AwsLookupUtil> awsLookupUtil,
-                       Provider<CachingSchemaUtil> cachingSchemaUtil) {
+                       Provider<CachingSchemaUtil> cachingSchemaUtil,
+                       DefaultAccountCredentialsProvider accountCredentialsProvider) {
```

Comment:
Why `DefaultAccountCredentialsProvider` and not `AccountCredentialsProvider`?

Reply:
Updated to use `AccountCredentialsProvider` instead